Cambios a la creación de baterias

Evankhell561 opened the pull request:
**What does this PR do:**
-las baterias creadas en el autlathe o en el protolathe inician con cero de carga pero se crean más rapido (a la misma velocidad que los otros stocks parts).
-añade la high cell+ descargada al code
-añade la xenobluecell descargada al code
-la super celda requiere tecnologia 4
-añade el diseño de la high cell+ al protolathe por tecnologia 3
-elimina nombres de pj de terceros de la xenoblue cell
-al crear una xenobluecell esta aparece en tus manos en vez de en el suelo
-añade el diseño de la xenobluecellmaker al podfabricator


Como ya le he explicado a daniel, este pr solo afecta a las baterias creadas en el autolathe y los protolahtes por esto, no afectará a casi a ningun departamento que no sea ciencias y más especificamente a robotica. Los ingenieros por su parte tienen un reserva de baterias de alta capacidad (que ya están cargadas) desde el inicio y nunca en todo mi tiempo en ciencias he visto a un ingeniero yendo a pedir baterias en ciencias. En cambio los unicos que usan las baterias hechas por ciencias son los robotistas ya que los cientificos de rnd usan sus baterias para mejorar maquinas y estas no requieren que la bateria tenga carga. 

**¿Por qué es bueno este pr?**
Este pr está enfocado en mejorar la experiencia de ciencias mediante un nerfeo a ciencias. ¿como es posible esto?
1. Los cientificos de rnd suelen no mejorar los cargadores de borgs y los mechas y aun si los mejoran nadie los usa y es que, ¿por qué poner a cargar un mecha por un minuto cuando el robotista ya puede cambiarle la bateria por una igual de potente y que puede cambiar en un segundo? esto le dará utilidad a esos cargadores y al hecho de mejorar esos cargadores por lo que los buenos cientificos los mejorarán más a menudo y tendrán un efecto más visible.

2. Los robotistas podrán usar todas las mecanicas de siempre para cambiar baterias rapidamente pero se sentirán en un ambiente más realista donde si alguien llega a tu taller para que le repares el motor del carro, no le vas a cambiar el motor dañado por uno nuevo sino que vas a repararlo. En este caso es tener más frecuentemente un mecha cargando en tu taller. También, para quien quiera una mejora en la bateria de su mecha tendrá que pagar el costo de dejarlo cargando un rato. ¿quieres una mejora en tu bateria? pues debes decidir si pagas el precio de si estás dispuesto a que tu nueva y más potente bateria se cargue o si sigues con la vieja

**Changelog:**
:cl:Evankhell
add: añade la high cell+ descargada al codigo y al protolahte. 
teak: la super cell requiere más tecnologia.
add: añade la xenobluecell descargada al codigo.
balance: las baterias creadas en el protolathe/autolathe inician descargadas pero se crean más rapido.
fix: arregla la ruta de la xenobluespacepowercell.
balance: la xenobluecell inicia descargada al ser crafteada pero se carga más rapido.
del: elimina nombres de pj de terceros de la xenobluespacepowercellmaker.
tweak: la xenobluecellmaker se puede crear en la en el pod fabricator.
tweak: al crear la xenobluecell, ésta aparece en tu mano en vez de en el suelo.
/:cl:

Commit:
cambios a la xenoblue cell

-la xenoblue cell inicia descargada pero carga más rapido
-mejora el codigo que hace que se cree en tu mano.

Commit:
Las baterias del protolathe/autolathe inician descargadas

-las baterias creadas en el autlathe o en el protolathe inician con cero de carga
-la carga máxima de los mechas se ajusto al tipo de bateria que usa
-la carga de los mechas al ser creados es menor que la caarga máxima de su bateria

UmeBelle commented:
Why 
Es un dolor tener que cargar las baterias cada vez que imprimes una, sobre todo para inge si se trata de una emergencia donde se ha ido la electricidad

Dapple27 commented:
pienso igual, esto no deberia ser mergeado

Evankhell561 commented:
1. La mayoria de las baterias se utilizan es para crear maquinaria. En estos casos el hecho de que las baterias estén descargadas no tiene importancia. 
2. Estaria que se usaran un poco más los cargadores que antes, esto incluye el de los mechas y el de los borgs.
3. cuando se va la energia en muy contadas ocasiones es que se reemplaza la bateria de los apc

Commit:
revierte los cambios a las baterias de los mechas.

Commit:
nombres de pj de terceros eliminados

Dapple27 commented:
Bueno, despues de haber hablado contigo cambio mi opinion, esta bien este pr

Commit:
más cambios a la creacion de baterias

-las baterias se crean más rapido
-añade el diseño de la high plus cell
-sube el reqtech de la super cell
-la xenoblue cell se crea en tu mano

Evankhell561 commented:
Aclaro, de nuevo. Estos cambios solo afectan a las baterias creadas en el autolathe/protolathe. las que están en el mapa, los vending o las que se piden por cargo no serán afectadas.
